doc/classes/CameraFeed.xml

```diff
@@ -34,6 +34,19 @@
 				Returns the position of camera on the device.
 			</description>
 		</method>
+		<method name="set_format">
+			<return type="bool" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="parameters" type="Dictionary" />
+			<description>
+				Sets the feed format parameters for the given index in the [member formats] array. Returns [code]true[/code] on success.
+				By default YUYV encoded stream is transformed to FEED_RGB.
+				YUYV encoded stream output format can be changed with [param parameters].[code]output[/code] value:[br]
```

Comment:
What is the `[br]` here for?

Reply:
Removed them, documentation look better now.